src/internal_macros.rs

```diff
@@ -117,7 +117,6 @@ macro_rules! debug_from_display {
     };
 }
 pub(crate) use debug_from_display;
-
```

Comment:
Meh, empty line would be nicer here.

Reply:
Agreed, I did #1255 to make it a little bit better